SevenZip/ArchiveEmulationStreamProxy.cs

```diff
@@ -18,16 +18,19 @@ internal class ArchiveEmulationStreamProxy : Stream, IDisposable
         /// </summary>
         public Stream Source { get; }
 
+        readonly bool _leaveOpen = false;
```

Comment:
Once again, purely preference but please add a new line below this line to separate it from the following comment :)

Reply:
fixed